media/js/src/wizard.js

```diff
@@ -1,15 +1,52 @@
 ;(function() {
 
+    var allRadioButtonsSelected = function(radios) {
+        var groups = {};
+        radios.each(function(a) {
+            groups[$(this).attr('name')] = 1;
+        });
+        var total = Object.keys(groups).length;
+        var selected = radios.filter(function(index) {
+            return $(this).is(':checked');
+        });
+        return total == selected.size();
```

Comment:
Yeah -- I thought it did too. I assumed you didn't have it set up on this project, but you do. I'll look into why it's not catching this.